### ISSUE TITLE
Extinguisher grenade

### ISSUE THREAD

**Issue by XenonPPG** [studio-minus/ppg-bugs-and-requests#1039]
**Feature description**
Spawnable item. When activated extinguishes all fire in radius (~7m)

**Feature purpose**
This would be useful when you have to extinguish many burning objects (Extinguisher works good only on objects on top, items below is hard to extinguish (too many word extinguish))

**Additional context**
Possible sprite
![extinguisher grenade](https://user-images.githubusercontent.com/102830220/161332223-245269c1-eceb-4265-b641-43858953e5c4.png)

**Comment by DindinYT37** [comment]
Yes. Dunno about the sprite, doesn't look bad but who knows if it'll get taken.
